### PR TITLE
BUGFIX: Fix local folder structure of resources

### PR DESCRIPTION
**Pull request by dlubitz**
Fixes: #56 again

### CODE REVIEW

File: cmd/beach/cmd/resource-download.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func handleResourceDownloadRun(cmd *cobra.Command, args []string) {
 			log.Error(err)
 		} else {
 			source := bucket.Object(attributes.Name)
-			targetPath := filepath.Dir(filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name)))
+			targetPath := filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name))
```
Comment (kdambekalns) [inline cmd/beach/cmd/resource-download.go:116]: Would it make sense to leave `Dir` in place but join the filename "correctly", i.e. by adding a `/ ` in between? 🤔

Comment (dlubitz) [reply cmd/beach/cmd/resource-download.go:116]: I thought the same, but filepath.Join removes it.

Comment (dlubitz) [reply cmd/beach/cmd/resource-download.go:116]: https://pkg.go.dev/path/filepath#Join

Comment (dlubitz) [reply cmd/beach/cmd/resource-download.go:116]: Or do you mean 
```
targetPath := filepath.Dir(filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name), attributes.Name))
```

Comment (kdambekalns) [reply cmd/beach/cmd/resource-download.go:116]: Sorry for the delay, I had to look at the code in full. I got the issue now and have it fixed…

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**